src/Microsoft.HttpRepl.Tests/Fakes/FileSystemStub.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.HttpRepl.FileSystem;
+
+namespace Microsoft.HttpRepl.Tests.TestDoubles
+{
+    internal class FileSystemStub : IFileSystem
```

Comment:
This is kind of redundant with MockedFileSystem in the integration tests project (which, as we've discussed, probably needs to be moved back to unit tests at some point - integration tests shouldn't be using a fake).  Just pointing it out as something to cleanup when we do that merge.

Reply:
I edited my earlier reply after I initially posted it, so you probably missed the second half of it:  
> I imagine we might want to merge the implementations but figured I would leave that for when we get to discussing moving the current integration-tests-that-aren't into this assembly.